### PR TITLE
[Tizen.Network] Add Tizen.Network

### PR DESCRIPTION
**Pull request by chleun-moon**
### Description of Change ###
Add Tizen.Network namespace to contains enumerations commonly used in sub namespaces (Tizen.Network.xxx)

### API Changes ###
 - ACR: http://suprem.sec.samsung.net/jira/browse/TCSACR-257

Add new Enumerations to be used Tizen.Network.Connection, Tizen.Network.WiFi and Tizen.Network.NetworkMonitor (to be added newly).

### CODE REVIEW

File: src/Tizen.Network/Tizen.Network/NetworkEnumerations.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ * limitations under the License.
+*/
+
+namespace Tizen.Network
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+	public enum ConnectionType
```
Comment (myroot) [inline src/Tizen.Network/Tizen.Network/NetworkEnumerations.cs]: It is duplication of Tizen.Network.Connection.ConnectionType

File: src/Tizen.Network/Tizen.Network/NetworkEnumerations.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		/// </summary>
+		/// <since_tizen> 6 </since_tizen>
+		NetProxy,
+	}
+
+    /// <summary>
+    /// Enumeration for the proxy method type.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public enum ProxyType
```
Comment (myroot) [inline src/Tizen.Network/Tizen.Network/NetworkEnumerations.cs]: It is duplication of Tizen.Network.Connection.ProxyType

File: src/Tizen.Network/Tizen.Network/NetworkEnumerations.cs (hunk 1 above)
Comment (chleun-moon) [reply src/Tizen.Network/Tizen.Network/NetworkEnumerations.cs]: Yes. ConnectionType and ProxyType are defined in Tizen.Network.Connection. They are used in Tizen.Network.WiFi and will be used in Tizen.Network.NetworkMonitor to be newly added. So we want to add ConnectionType and ProxyType in Tizen.Network namespace and deprecate from Tizen.Network.Connection namespace. What do you think about it?

Comment (myroot) [reply src/Tizen.Network/Tizen.Network/NetworkEnumerations.cs]: I don't recommend it, it cause too many API changes.
If Tizen.Network.Connection.ConnectionType is deprecated, the following APIs are also deprecated
`public static ConnectionState WiFiState`
`public static ConnectionState BluetoothState`
`public ConnectionState State`

I recommend to use Tizen.Network.Connection.ConnectionType in othres

Comment (chleun-moon) [reply src/Tizen.Network/Tizen.Network/NetworkEnumerations.cs]: Ok. I'll cancel this PR as your recommend.
Thanks.

**Pull request closed without merging**